client/src/components/body/profilepage/Profile.js

```diff
@@ -49,34 +48,38 @@ const Profile = () => {
   }
 
   return (
-    <React.Fragment>
+    <div className={"tw-mt-[3rem]"}>
       {state.main.user?.firstname === null ? (
         <h3>You are currently not logged in.</h3>
       ) : (
-        <div className="profile container">
+        <div className="tw-mt-0 tw-w-full">
           <ProfileHeader
-            user={state.main.user}
+            user={user}
             labRecords={labRecords}
             toDoLabs={toDoLabs}
```

Comment:
It doesn't show well in this screenshot but my profile picture is very blurry

![Screenshot 2024-10-25 at 1 07 20 PM](https://github.com/user-attachments/assets/f4a99517-b6d4-4d52-b9e8-4a26126e746a)

Reply:
I believe the issue is that google's image they present are very small, and it's trying to stretch the image to fit the proper size.